src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/UserControl.xaml

```diff
@@ -0,0 +1,26 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+    <Style x:Key="DefaultUserControlStyle" TargetType="{x:Type UserControl}">
+        <Setter Property="Border.CornerRadius" Value="4" />
```

Comment:
Similarly for here, shall we remove CornerRadius default value here to keep it unbiased for this control ?